components/common/itemTransfer/ItemTransferModal.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div>
+    <Loader v-model="isLoading" :status="status" />
+    <NeoModal v-model="isModalActive" scroll="clip" @close="onClose">
+      <div class="modal-width">
+        <header
+          class="border-bottom border-grey is-flex is-align-items-center is-justify-content-space-between px-6">
+          <p class="py-5 is-size-6 has-text-weight-bold">
+            {{ $t('transaction.transferNft') }}
+          </p>
+
+          <NeoButton
+            variant="text"
+            no-shadow
+            icon="xmark"
+            icon-pack="fa-sharp"
+            size="medium"
+            class="cross"
+            @click.native="onClose" />
+        </header>
+
+        <div class="px-6 pt-4 pb-5">
+          <ModalIdentityItem />
+
+          <div class="mt-4 is-flex is-justify-content-space-between">
+            <div class="is-flex">
+              <div>
+                <BasicImage
+                  :src="avatar"
+                  :alt="nft?.name"
+                  class="border image is-48x48" />
+              </div>
+
+              <div
+                class="is-flex is-flex-direction-column is-justify-content-space-between ml-4 limit-width">
+                <div
+                  class="has-text-weight-bold has-text-color line-height-1 no-wrap is-clipped is-ellipsis">
+                  {{ nft.name }}
+                </div>
+                <div class="line-height-1 no-wrap is-clipped is-ellipsis">
+                  {{ nft.collection?.name || nft.collection.id }}
+                </div>
+              </div>
+            </div>
+
+            <div class="is-flex is-align-items-end">
+              {{ nftPrice }}
+            </div>
+          </div>
+
+          <hr class="my-4" />
+
+          <h2 class="mb-2 has-text-weight-bold has-text-color is-capitalized">
+            {{ $t('transaction.transferTo') }}
+          </h2>
+
+          <AddressInput
+            v-model="address"
+            :is-invalid="isYourAddress"
+            label=""
+            class="is-flex-1"
+            placeholder="Enter wallet address"
+            with-address-check
+            @check="handleAddressCheck" />
+        </div>
+
+        <div class="px-6 pb-4 is-flex is-flex-direction-column">
+          <NeoButton
+            :disabled="isDisabled"
+            :label="transferItemLabel"
+            variant="k-accent"
+            no-shadow
+            class="is-flex is-flex-grow-1 py-5 is-capitalized btn-height"
```

Comment:
have you tried size prop instead of specific class? I think you can use `size="large"`

Reply:
yes , makes the text bigger and also the button height is not the one requested 
<img width="377" alt="CleanShot 2023-09-12 at 12 10 17@2x" src="https://github.com/kodadot/nft-gallery/assets/44554284/3f14018f-6649-4976-a8b4-6811b55b3c4d">
<img width="365" alt="CleanShot 2023-09-12 at 12 10 53@2x" src="https://github.com/kodadot/nft-gallery/assets/44554284/42dc973d-2373-45a2-86f6-74fb6aa80bf8">

should be
<img width="390" alt="CleanShot 2023-09-12 at 12 11 11@2x" src="https://github.com/kodadot/nft-gallery/assets/44554284/c9aa0f13-f52e-45ce-b43d-24f5a9930241">

just trying to follow how it's already done in other places

Reply:
alright I'll open following issue for buttons size